mantle/platform/local/cluster.go

```diff
@@ -37,23 +37,14 @@ func (lc *LocalCluster) NewCommand(name string, arg ...string) exec.Cmd {
 	return cmd
 }
 
-func (lc *LocalCluster) hostIP() string {
-	// hackydoo
-	bridge := "br0"
-	for _, seg := range lc.flight.Dnsmasq.Segments {
-		if bridge == seg.BridgeName {
-			return seg.BridgeIf.DHCPv4[0].IP.String()
-		}
-	}
-	panic("Not a valid bridge!")
-}
-
 func (lc *LocalCluster) NewTap(bridge string) (*TunTap, error) {
 	nsExit, err := ns.Enter(lc.flight.nshandle)
 	if err != nil {
 		return nil, err
 	}
-	defer nsExit()
+	defer func() {
+		_ = nsExit()
```

Comment:
This doesn't really fix the fact that the error return value is not checked. It would be better to check the error and forward the non-nil case. That would likely require naming the error return value for the function, and setting it in this defer block.

Reply:
Thank you again for review. Fixed.